### DISCUSSION TITLE
Superliminal

### DISCUSSION THREAD

**Discussion by dursonATWIT** [elrumo/macOS_Big_Sur_icons_replacements#1182] [Icon request] [answered]
I tried to think of an edit for it myself, but the best I can come up with is a boring white background. Thanks for any help!
![Superliminal](https://user-images.githubusercontent.com/71397056/119597742-a2038c00-bdaf-11eb-9281-284bdab01a2e.png)

**Comment by cake03** [comment, marked as answer]
<a href="https://user-images.githubusercontent.com/80169337/122049056-9c71f280-cdff-11eb-98f7-8b006bfc3766.png">
<img src="https://user-images.githubusercontent.com/80169337/122049056-9c71f280-cdff-11eb-98f7-8b006bfc3766.png" width="512"/>
</a>
